### PR TITLE
Noamlevy/sc 16173/bump to grafana 11 gc changes

### PR DESCRIPTION
**Pull request by noamApps**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -188,3 +189,21 @@ COPY ${RUN_SH} /run.sh
...
+
+COPY --from=go-src /tmp/grafana/bin/grafana* /tmp/grafana/bin/*/grafana* ./bin/
+COPY --from=js-src /tmp/grafana/public ./public
+
+USER 0
+
+ENV GF_PLUGIN_DIR="/usr/share/grafana/plugins" \
+    GF_PATHS_PLUGINS="/usr/share/grafana/plugins"
+
+ENV GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=grafana-clickhouse-datasource
```
Comment (steb-veksler) [inline Dockerfile]: can remove, we use original version

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: arm64
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      -
+        name: Get Grafana Version
+        run: echo "GF_VERSION=$(npm pkg get version | tr -d '"')" >> $GITHUB_ENV
```
Comment (steb-veksler) [inline .github/workflows/pr.yaml:41]: fail on CI - https://github.com/groundcover-com/grafana/actions/runs/9445031278

File: Dockerfile [hunk 3]
```diff
@@ -38,14 +40,11 @@ ARG COMMIT_SHA=""
 ARG BUILD_BRANCH=""
 ARG GO_BUILD_TAGS="oss"
 ARG WIRE_TAGS="oss"
-ARG BINGO="true"
-
-# This is required to allow building on arm64 due to https://github.com/golang/go/issues/22040
-RUN apk add --no-cache binutils-gold
+ARG BINGO="false"
```
Comment (steb-veksler) [inline Dockerfile]: try to use bingo

**Review by steb-veksler — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (noamApps) [reply Dockerfile]: done

File: Dockerfile (hunk 1 above)
Comment (noamApps) [reply Dockerfile]: done

**Pull request merged**